src/js/outreach/views/login_views.js

```diff
@@ -19,9 +19,22 @@ const LoginView = View.extend({
   template: hbs`
     <div class="dialog__icon">{{far "lock-keyhole"}}</div>
     <div>Enter your date of birth to access this form.</div>
-    <div><input type="date" class="js-date dialog__input{{#if hasError}} has-error{{/if}}" required pattern="\d{4}-\d{2}-\d{2}" placeholder="Your Date of Birth" value="{{ dob }}"></div>
-    {{#if hasError}}<div class="dialog__error">That date of birth does not match our records. Please try again.</div>{{/if}}
-    <div><button class="button--green dialog__button js-submit" {{#unless dob}}disabled{{/unless}}>Continue to Form {{fas "right-to-bracket"}}</button></div>
+    <div>
```

Comment:
don't need to change this right now

Reply:
Oh forgot to change that back. I added spacing and line breaks so I could read the code easier when I was copying stuff from it.